src/apis/computed.ts

```diff
@@ -9,7 +9,7 @@ interface Option<T> {
 }
 
 // read-only
-export function computed<T>(getter: Option<T>['get']): Readonly<Ref<T>>;
+export function computed<T>(getter: Option<T>['get']): Readonly<Ref<Readonly<T>>>;
 // writable
 export function computed<T>(options: Option<T>): Ref<T>;
```

Comment:
Considering how computed works, the return value of a writable computed should be `Ref<readonly<T>>` instead of Ref<T>.

Reply:
Thx review! Yeah, I think so.
So how about this change?
https://github.com/vuejs/composition-api/pull/145/commits/4fbfdcfed3a4fb592b5707d4594f48a25b258149